docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+services:
+  nginx:
+    image: bitnami/nginx
+    volumes:
+      - /home/debian/ircd2/nginx/conf/private/server_block.conf:/opt/bitnami/nginx/conf/server_blocks/my_server_block.conf:ro
```

Comment:
Doesn't this mean there needs to be a user account called "debian" created on the host machine?

I think it would be better to define the volume with a relative path within the repo

Reply:
No - my original idea was to have users of the repo edit those paths to fit their machines (ie /home/ski/repo for me), but I've updated docker-compose with relative paths, I agree, that is a better decision.